QgisModelBaker/gui/workflow_wizard/workflow_wizard.py

```diff
@@ -336,15 +336,17 @@ def id_changed(self, new_id):
             models = []
             datasets = []
             baskets = []
-            if self.current_export_filter == SchemaDataFilterMode.MODEL:
-                models = self.export_models_model.checked_entries()
-            elif self.current_export_filter == SchemaDataFilterMode.DATASET:
-                datasets = self.export_datasets_model.checked_entries()
-            elif self.current_export_filter == SchemaDataFilterMode.BASKET:
-                baskets = self.export_baskets_model.checked_entries()
+            if self.current_filter_mode == SchemaDataFilterMode.MODEL:
+                models = self.current_models_model.checked_entries()
+            elif self.current_filter_mode == SchemaDataFilterMode.DATASET:
+                datasets = self.current_datasets_model.checked_entries()
+                print(datasets)
```

Comment:
It's a forgotten debug print or it's intended to stay?

Reply:
Debug print... Thanks for spotting.

Reply:
```suggestion
```